ChooseArea.svelte

```diff
@@ -9,8 +9,11 @@
 
   let showAbout = false;
 
-  let inputList;
+  let inputValue;
   let dataList;
+  let authoritySet = new Set();
+
+  $: validEntry = authoritySet.has(inputValue);
```

Comment:
https://svelte.dev/tutorial/reactive-declarations makes it trivial for us to recalculate this when the input changes